linkerd/protocol/h2/src/test/scala/io/buoyant/linkerd/protocol/h2/H2ClassifiersTest.scala

```diff
@@ -167,19 +166,4 @@ class H2StreamClassifiersTest extends FunSuite {
       }
     }
   }
-
-  test("NonRetryableStream: makes RetryableFailures NonRetryableFailures when chunked") {
```

Comment:
Is there a reason these tests were removed instead of being rewritten to use stream classifiers?